load/Validate.py

```diff
@@ -78,7 +78,12 @@ def process(self, filesets):
 		#find.moveDuplicates(duplicates)
 
 	def validateTextPlainFilesets(self, texts, inp, filePath):
-		errorTuple = texts.validateFileset("text_plain", inp.bibleId, inp.filesetId, inp.languageRecord, inp.index, filePath)
+		subTypeCode = "text_plain"
+
+		if inp.subTypeCode() == "text_plain":
```

Comment:
if we have a loading text with a filesetid directory (directory doesn't contain the USX word) we should specify that the `subTypeCode` is equals to `text_usx`